analyzer/protobuf.spicy

```diff
@@ -18,14 +18,31 @@ type WireType = enum {
     VARINT = 0x00,
     I64    = 0x01,
     LEN    = 0x02,
-    SGROUP = 0x03, # deprecated
-    EGROUP = 0x04, # deprecated
+    SGROUP = 0x03, # deprecated, but remains in the wire format.
+    EGROUP = 0x04, # deprecated, but remains in the wire format.
     I32    = 0x05
 };
 
+const UNSET_GROUP: int32 = -1;
```

Comment:
I couldn't find a way to track groups for fields sandwiched between `SGROUP` and `EGROUP` without using a global variable. Is there such thing as statics in spicy? Suggestions welcome. :)

Reply:
Off the top of my head, I am not aware of another way to approach this without using a global variable.